scrapper.py

```diff
@@ -35,7 +44,29 @@ def find_articles(self):
         """
         Finds articles
         """
-        pass
+        # html = "https://languagejournal.spbu.ru/issue/view/682"
+        user_agent = "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:99.0) " \
+                     "Gecko/20100101 Firefox/99.0"
+        accept = "*/*"
+        accept_encoding = "gzip, deflate, br"
+        accept_language = "en-US,en;q=0.5"
+        headers = \
+            {
+                'User-Agent': user_agent,
+                'Accept': accept,
+                'Accept-Encoding': accept_encoding,
+                'Accept-Language': accept_language
+            }
+        for seed_url in self.seed_urls:
+            response = requests.get(seed_url, headers=headers)
+            if not response.ok:
+                print("Failed")
+
+        soup = BeautifulSoup(response.text, 'lxml')
```

Comment:
are you really going to parse the content of a single seed? or we want to analyze each seed?

Reply:
no, sorry